### DISCUSSION TITLE
Support for RedisGraph

### DISCUSSION THREAD

**Discussion by SoulPancake** [redis/go-redis#2496] [Ideas]
Just like Jedis has the support for this and the commands,
We can also start a project on this
Is there any such plan in the way ahead for this?

**Comment by monkey92t** [comment]
Yes, go-redis accepts any redis related modules, if you are interested, you can describe your ideas in detail :)

**Reply by SoulPancake** [reply]
@monkey92t Can we start a tracking issue for all the RedisGraph commands and then we start with the decisions on the API design beginning from the configs ?
